assembler/src/common/sequence/seq_common.hpp

```diff
@@ -15,6 +15,7 @@
 #ifndef SEQ_COMMON_HPP_
 #define SEQ_COMMON_HPP_
 
+#include <stdint.h>
```

Comment:
Thanks.

First of all, it should be cstdint as this is C++ header. And the system include must be after the local (`k_range.hpp`) ones.

Reply:
Sure, thanks, changed to `<cstdint>` and moved after local `#include` statement. I also changed `u_int64_t` to `uint64_t` for portability (`uint64_t` is the standard) in a commit below